boards/BMS/Src/Io/Io_LTC6813/Io_LTC6813Shared.c

```diff
@@ -170,14 +170,18 @@ static void Io_PrepareCfgRegBytes(
     Io_LTC6813CellVoltages_GetMinCellLocation(
         &min_cell_segment, &min_cell_index);
 
-    // Write to the last segment on the daisy chain first
-    for (uint8_t curr_segment = NUM_OF_ACCUMULATOR_SEGMENTS - 1U;
-         (int8_t)curr_segment >= ACCUMULATOR_SEGMENT_0; curr_segment--)
+    // Write to the configuration registers of each segment
+    for (uint8_t curr_segment = 0;
+         (int8_t)curr_segment < NUM_OF_ACCUMULATOR_SEGMENTS; curr_segment++)
     {
+        // Configuration registers are written to serially starting from the
+        // last segment
+        const int32_t tx_cfg_idx =
```

Comment:
why use an int32?

Reply:
we should use a uint8

Reply:
I was getting compiler warnings trying to get it to `uint8_t` but it's now been resolved lol. See latest commit